src/hash/rpo/tests.rs

```diff
@@ -200,6 +202,18 @@ fn hash_test_vectors() {
     }
 }
 
+#[test]
+fn sponge_bytes_with_remainder_length() {
+    Rpo256::hash(&vec![0_u8; 111]);
+}
```

Comment:
Is this also just testing that it won't panic? Can you add a comment that explains what this test is for?

Reply:
Indeed, the number 111 is quite "magic" here. Comment added; I also changed the number to a prime so we have the guarantee it won't be divisible.